src/m365/pa/commands/app/app-consent-set.ts

```diff
@@ -0,0 +1,93 @@
+import { Logger } from '../../../../cli/Logger';
+import GlobalOptions from '../../../../GlobalOptions';
+import request from '../../../../request';
+import { powerPlatform } from '../../../../utils/powerPlatform';
+import { validation } from '../../../../utils/validation';
+import PowerAppsCommand from '../../../base/PowerAppsCommand';
+import commands from '../../commands';
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  environment: string,
+  name: string;
+  bypass: boolean;
+}
+
+class PaAppConsentSetCommand extends PowerAppsCommand {
+  public get name(): string {
+    return commands.APP_CONSENT_SET;
+  }
+
+  public get description(): string {
+    return 'Makes sure users can bypass the API Consent window for the selected canvas app';
+  }
+
+  constructor() {
+    super();
+
+    this.#initOptions();
+    this.#initValidators();
+    this.#initTypes();
+  }
+
+  #initOptions(): void {
+    this.options.unshift(
+      {
+        option: '-e, --environment <environment>'
+      },
+      {
+        option: '-n, --name <name>'
+      },
+      {
+        option: '-b, --bypass <bypass>'
+      }
+    );
+  }
+
+  #initValidators(): void {
+    this.validators.push(
+      async (args: CommandArgs) => {
+        if (!validation.isValidGuid(args.options.name)) {
+          return `${args.options.name} is not a valid GUID`;
+        }
+
+        return true;
+      }
+    );
+  }
+
+  #initTypes(): void {
+    this.types.boolean.push('bypass');
+  }
+
+  public async commandAction(logger: Logger, args: CommandArgs): Promise<void> {
+    if (this.verbose) {
+      logger.logToStderr(`Setting the bypass consent for the Microsoft Power App ${args.options.name}... to ${args.options.bypass}`);
+    }
+
+    const dynamicsApiUrl = await powerPlatform.getDynamicsInstanceApiUrl(args.options.environment, args.options.asAdmin);
```

Comment:
We don't have an `asAdmin` option in this command. Do you know if that would be an interesting addition? Would it work in both scenario's?